apps/src/templates/rubrics/AiAssessment.jsx

```diff
@@ -5,14 +5,15 @@ import style from './rubrics.module.scss';
 import {Heading6} from '@cdo/apps/componentLibrary/typography';
 import AiAssessmentBox from './AiAssessmentBox';
 import AiAssessmentFeedback from './AiAssessmentFeedback';
+import {aiEvaluationShape} from './rubricShapes';
 const icon = require('@cdo/static/ai-bot.png');
 
 export default function AiAssessment({
   isAiAssessed,
   studentName,
   aiUnderstandingLevel,
   aiConfidence,
-  learningGoalKey,
+  aiInfo,
```

Comment:
can we please call this `aiEval` or `aiEvalInfo`, to make it a bit more specific and to more closely match the name of aiEvaluationShape?

Reply:
also, any reason we still need aiUnderstandingLevel and aiConfidence as props, now that we have the aiEvaluation, which contains both of these? https://github.com/code-dot-org/code-dot-org/blob/6c6897ec21ee803689035f609983c633beb73b5f/apps/src/templates/rubrics/rubricShapes.jsx#L45-L49

if not, you could get rid of those props, and do something like this here:

```js
const aiConfidence = aiEval.ai_confidence;
...
```